jenkins/deployment/community/cortx-community-deploy.groovy

```diff
@@ -79,7 +79,7 @@ pipeline {
                         AWS_IP=$(terraform show -json terraform.tfstate | jq .values.outputs.cortx_deploy_ip_addr.value 2>&1 | tee ip.txt)
                         IP=$(cat ip.txt | tr -d '""')                        
                         ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@${IP} sudo bash /home/centos/setup.sh
-                        sleep 120
+                        sleep 240
```

Comment:
Can we add some status check instead of arbitrary sleep

Reply:
After running the steup.sh the instance reboot actually  and failing the step. So the logics are getting bypassed.